### ISSUE TITLE
Using sort on a search index no longer works in 1.6.1

### ISSUE THREAD

**Issue by rws-github** [cloudant/nodejs-cloudant#163]
If I use 1.6.1 and have a sort parameter of `since<number>` when using a search query, I get a 400 - bad request from Cloudant. If I use 1.5.2 it works just fine.

**Comment by glynnbird** [comment]
Please could you post a code snipped where you are calling the .search function?

I suspect it is because we are no longer JSON.stringifying certain parameters:

```
db.search('enhanced','search', {q: 'trump', sort:JSON.stringify('title<string>')}, function(err, data) {
  console.log(err,data);
})
```

I added code to prevent 'double-stringification' of some parameters.

**Comment by rws-github** [comment]
params.sort = 'since<number>'
db.search(applicationDesignDoc, 'applications', params)

**Comment by glynnbird** [comment]
ok thanks. I can fix this pretty simply. I'll post a fix later today.

**Comment by rws-github** [comment]
Your suggestion does work:
```
params.sort = JSON.stringify('since<number')
```

**Comment by glynnbird** [comment]
Fixed in v1.6.2
